e2e-test/cypress/integration/groups_spec.js

```diff
@@ -43,4 +43,26 @@ describe("User/Groups Dashboard", function() {
         expect($th.first()).to.have.text("TestGroup");
       });
   });
+
+  it("User who created the group should be able to edit it", function() {
+    cy.get("[data-test=edit-group-TestGroup]").should("be.visible");
+  });
+
+  it("Other users should not be able to edit the group", function() {

```

Comment:
"Other" users are users that have not created the group - the test description should be more clear